src/histolab/slide.py

```diff
@@ -200,7 +200,7 @@ def locate_mask(
 
     @lazyproperty
     def name(self) -> str:
-        """Retrieve the slide name without extension.
+        """Return the slide name without extension.
```

Comment:
A property doesn't compute any action it's a property and for this the docstring should be something without the verbs. I'd use Slide name without extension.